components/vf-search/vf-search.config.yml

```diff
@@ -1,34 +1,89 @@
 title: Search
 label: Search
-status: beta
+status: alpha
+preview: '@preview--nogrid'
 
 context:
   component-type: block
 
 variants:
   - name: default
-    label: Default
-    hidden: false
+    label: Search
+    hidden: true
+
+  - name: responsive
+    label: Responsive Search
+    context:
+      responsive: true
+      search_action: '#'
+      search_autofocus: false
+      search_button: Search
+      search_label: Search
+      search_placeholder: Enter your search terms
+      search_id: searchitem
+      search_value_default: ''
+
+  - name: responsive results
+    label: Responsive with Results
+    context:
+      results: true
+      responsive: true
+      search_action: '#'
+      search_autofocus: false
+      search_button: Search
+      search_label: Search
+      search_placeholder: Enter your search terms
+      search_id: searchitem
+      search_value_default: ''
+
+  - name: mini
+    label: Mini Search
+    context:
+      mini: true
+      search_action: '#'
+      search_autofocus: false
+      search_button: Search
+      search_label: Search
+      search_placeholder: Enter your search terms
+      search_id: searchitem
+      search_value_default: ''
+
+  - name: mini results
+    label: Mini with Results
+    context:
+      mini: true
+      results: true
+      search_action: '#'
+      search_autofocus: false
+      search_button: Search
+      search_label: Search
+      search_placeholder: Enter your search terms
+      search_id: searchitem
+      search_value_default: ''
+
+  - name: container
+    label: Search Container
     context:
-      variant: default
+      responsive: true
+      search__background: "#F3F3F3"
```

Comment:
Would it be better to pass in one of the css custom props that has a design token value? 

I think this would work?

```
      search__background: "var(--vf-ui-color--grey--light,#F3F3F3)"
```

Reply:
Ah, yes - that might be better. I'll take a look and update.